src/endpoints/extensions.js

```diff
@@ -224,12 +224,20 @@ router.post('/version', jsonParser, async (request, response) => {
             return response.status(404).send(`Directory does not exist at ${extensionPath}`);
         }
 
+        let currentCommitHash;
+        try {
+            currentCommitHash = await git.cwd(extensionPath).revparse(['HEAD']);
+        } catch (error) {
+            // it is not a git repo, or has no commits yet, or is a bare repo
+            // not possible to update it, most likely can't get the branch name either
+            return response.send({ currentBranchName: null, currentCommitHash, isUpToDate: true, remoteUrl: null });
```

Comment:
this is on the basis of isUpToDate being used as ! "update available" in the UI, and an update is not available so we are just returning true facts